docs/core/diagnostics/dotnet-gcdump.md

```diff
@@ -0,0 +1,110 @@
+---
+title: dotnet-gcdump - .NET Core
+description: Installing and using the dotnet-gcdump command-line tool.
+ms.date: 07/26/2020
+---
+# Heap analysis tool (`dotnet-gcdump`)
+
+**This article applies to:** ✔️ .NET Core 3.1 SDK and later versions
+
+## Installing `dotnet-gcdump`
+
+To install the latest release version of the `dotnet-gcdump` [NuGet package](https://www.nuget.org/packages/dotnet-gcdump), use the [dotnet tool install](../tools/dotnet-tool-install.md) command:
+
+```dotnetcli
+dotnet tool install -g dotnet-gcdump
+```
+
+## Synopsis
+
+```console
+dotnet-gcdump [-h|--help] [--version] <command>
+```
+
+## Description
+
+The `dotnet-gcdump` global tool is a way to collect gcdumps of live .NET processes. It is built using the EventPipe technology which is a cross-platform alternative to ETW on Windows. Gcdumps are created by triggering a GC in the target process, turning on special events, and regenerating the graph of object roots from the event stream. This allows for gcdumps to be collected while the process is running with minimal overhead. These dumps are useful for several scenarios:
```

Comment:
gcdumps/gcdumps isn't a word
```suggestion
The `dotnet-gcdump` global tool is a way to collect dumps of live .NET processes. It uses the EventPipe technology, which is a cross-platform alternative to ETW on Windows. Dumps are created by triggering a garbage collection (GC) in the target process, turning on special events, and regenerating the graph of object roots from the event stream. This allows for dumps to be collected while the process is running, with minimal overhead. These dumps are useful for several scenarios:
```
Can EventPipe technology be a link to an overview of that?

Reply:
Would that make any confusion with the `dotnet-dump` tool?

I'm not familiar with any of both the tools, but I think instead of replacing gcdumps with dumps, we may change it to gc dumps or GC dumps?

Reply:
The gcdump tool produces dumps -- in context there should be no cause for confusion because we're not talking about the dump tool. That said, if you prefer to use GC dumps I'm OK with that as long as at the first occurrence of GC the acronym is spelled out.